### PR TITLE
Document how to control the partition count

### PR DESCRIPTION
**Pull request by BGehrels**
The documentation does not specify how to choose the number of partitions when creating an event. This PR documents this process, and gives an example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,12 @@ The `schema` value should only declare the custom part of the event - the generi
 schema is implicit and doesn't need to be defined. The combination of the two 
 (the "effective schema") will be checked when events are submitted for the event type.
 
+Each event type can have a `default_statistic` object attached. It controlls the
```
Comment (tmuehl) [inline README.md]: controls (only `l`)

File: README.md [hunk 2]
```diff
@@ -171,6 +171,12 @@ The `schema` value should only declare the custom part of the event - the generi
...
+number of partitions of the underlying topic. If you do not provide any value, 
+Nakadi will use a sensible default value which may be just a single partition.
+This will effectively disallow parallel reads of subscriptions of this event
+type. The values provided here can not be changed later, so choose them wisely.
```
Comment (lmontrieux) [inline README.md]: can not -> cannot

Comment (BGehrels) [inline README.md]: > Both cannot and can not are acceptable spellings, but the first is much more usual.
https://en.oxforddictionaries.com/usage/cannot-or-can-not

**Comment by BGehrels** [comment]
Fine like that?

**Comment by lmontrieux** [comment]
@BGehrels thanks a lot, looks great to me. Could I ask you to edit your PR description, so it contains the motivation for the request? We need to keep track of the reason for each change, for compliance. I suggest something like this (feel free to copy/paste and modify if you want):

The documentation does not specify how to choose the number of partitions when creating an event. This PR documents this process, and gives an example.

**Comment by BGehrels** [comment]
Done

**Comment by lmontrieux** [comment]
:shipit:

**Comment by lmontrieux** [comment]
Thank you for your contribution. Nakadi's master branch has changed since you submitted your PR, would you mind updating your PR? Travis-CI won't build your PR, and I won't be able to merge it, until then.

**Comment by BGehrels** [comment]
Done

**Comment by lmontrieux** [comment]
👍

**Pull request merged**